### ISSUE TITLE
On click getting the wind speed along with Lat Long

### ISSUE THREAD

**Issue by arnica04** [Esri/wind-js#18]
Can we have something similar to what earth nullschool (http://earth.nullschool.net/ ) has. On click they display the Lat, Long along with the wind speed and direction.

**Comment by awaisaltaf** [comment]
Hi,
I have a similar question, can we get direction and speed like http://earth.nullschool.net/.

**Comment by rinku16** [comment]
@awaisaltaf yes you can get the code of (http://earth.nullschool.net/) in github here-(https://github.com/cambecc/earth) but i think it is limited in functionality as compare to demo link.

I am having the problem of not getting the temperature information along the wind speed and longitude and latitude information.

As shown below there is "no data" message display on click of tempeature.

![image](https://user-images.githubusercontent.com/31231671/67941050-e2fae500-fbfa-11e9-94bb-66a6dbc6032f.png)
